### ISSUE TITLE
Can't onboard with existing atsign in buzz

### ISSUE THREAD

**Issue by yahu1031** [atsign-foundation/at_client_sdk#656]
**Describe the bug**
I have onboarded with an atsign (`@mangotangostable`). But whenever I restarted my code, I can't auto onboard to the app.

**To Reproduce**
- Clone the latest buzz app from [here](https://github.com/ramtulluri/contacts-app/tree/atbuzz-2.0).
- Run the code and onboard with an atsign.
- Try hot-restarting the app.

**Expected behavior**
The app must auto onboard the app.

**Screenshots**

<img width="867" alt="Screenshot 2022-08-14 at 2 24 23 PM" src="https://user-images.githubusercontent.com/35523357/184529675-3e14c0c7-b506-403b-843a-23cb0c930af0.png">

**Smartphone (please complete the following information):**
 - Device: iPhone12, macOS
 - OS: iOS15.5, macOS12.5

**Were you using an @‎application when the bug was found?**
  - atbuzz

**Additional context**
I thought it was from onboarding but when I called respective methods from the KeychainManager, I got those results mentioned in the image above.

dependencies I use
```yaml
dependencies:
  at_onboarding_flutter: ^4.0.4
  at_lookup: ^3.0.27
  at_sync_ui_flutter: ^1.0.4
  at_commons: ^3.0.9
  at_contact: ^3.0.5
dependency_overrides:
  permission_handler: ^10.0.0
  at_client_mobile:
    git:
      url: https://github.com/atsign-foundation/at_client_sdk
      ref: 0909dcf
      path: at_client_mobile
  at_location_flutter:
    git:
      url: https://github.com/atsign-foundation/at_widgets.git
      ref: toast_fix
      path: at_location_flutter
```

**Comment by yahu1031** [comment]
I have deep-dived into the code and found something. So the direct use of `KeyChainManager.getInstance().getAtSign()` and `KeyChainManager.getInstance().getAtsignsWithStatus()` don't give the expected values where as `KeychainUtils.getAtSign()` and `KeychainUtils.getAtsignsWithStatus()` gives the expected results. In this case, We must need a little good docs WRT the changes mentioning not to use the old approach.

WRT the above understanding and work around, I would like to close this ticket as completed. But holding this back WRT documentation.

**Comment by gkc** [comment]
1. Are the old ways of things marked as deprecated?
2. If we are maintaining the "old ways", then their implementation needs to be changed so that internally they use the "new ways"

**Comment by yahu1031** [comment]
@gkc 
1. Nope they weren't deprecated. Instead, if we don't have any `KeyChainManager` usage and `KeychainUtil` replaces the `KeyChainManager`, We can make the `KeyChainManager` a private class. So all the functions cannot be accessed outside.
2. They have updated their approaches for storing keys WRT `biometric_storage` and `flutter_keychain`.

**Comment by yahu1031** [comment]
Now I can able to get values and can onboard the app automatically WRT the [hotfix changes](https://github.com/atsign-foundation/at_client_sdk/compare/trunk...hotfix/get-atsign-return-null).

**Comment by sachins-geekyants** [comment]
Moving this ticket to PR 55 .

**Comment by sonle-geekyants** [comment]
@sachins-geekyants  Please check and merge this PR to fix it: https://github.com/atsign-foundation/at_client_sdk/compare/trunk...hotfix/get-atsign-return-null

**Comment by sachins-geekyants** [comment]
Fix released with `at_onboarding_flutter 6.0.3 `
